<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>日常账本</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{background-image: url(:/new/myimage/new_icon/Background.jpg);}
QPushButton{	
	color:#000000;
	border:2px solid #ffffff;
	background-color:#bbcbd8;
	border-radius:5px;
}

QPushButton::hover{
	background-color:#52DCFE;
	color:#000000;
}

QPushButton::pressed,QPushButton::checked{
	background-color:#52DCFE;
	color:#000000;
}

QPushButton::disabled{
	background-color:transparent;
}
</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>420</width>
      <height>240</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>420</width>
      <height>240</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>420</width>
      <height>240</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>STKaiti</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="title">
     <string>总帐本</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>STKaiti</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;收入：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>STKaiti</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;支出：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>STKaiti</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;结余：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="income_count_all">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>STKaiti</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="outcome_count_all">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>STKaiti</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="netcome_count_all">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>STKaiti</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>420</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>420</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>420</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>STKaiti</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加一条新记账</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushbutoon{
	background-color: rgb(202, 215, 224);
	background-color:4CAF50;
	color:black;
	border-radius:25px;
}</string>
    </property>
    <property name="text">
     <string>记账数据统计</string>
    </property>
   </widget>
   <widget class="QLabel" name="Logo_image">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>-30</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="to_calendar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>按时间查找</string>
    </property>
   </widget>
   <widget class="QPushButton" name="all_history_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>54</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>全部历史收支</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
